### PR TITLE
[Project Submission] 3Web DAO

### PR DESCRIPTION
**Pull request by Jonkrua**
Submission 3Web DAO

### CODE REVIEW

File: projects/developer_tooling/3Web-DAO.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+
```
Comment (longfeiWan9) [inline projects/developer_tooling/3Web-DAO.md:1]: Can you please add a project name here?

File: projects/developer_tooling/3Web-DAO.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Category 
+
+infrastructure
```
Comment (longfeiWan9) [inline projects/developer_tooling/3Web-DAO.md:4]: This looks like an application for me other than developer toolings. Could you please update it category and move this markdown file to `application` folder?

File: projects/developer_tooling/3Web-DAO.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+https://3web.io/  (Coming soon...The domain name has been purchased and website is under development)
+
+## Docs
+
+https://mirror.xyz/0x8C5da2937C664882111FCfe717ca7AF02A6e6440
+
+## Team Info
+
+Team from https://github.com/3Web-DAO
```
Comment (longfeiWan9) [inline projects/developer_tooling/3Web-DAO.md:48]: Not anything related to your team

File: projects/developer_tooling/3Web-DAO.md (hunk 1 above)
Comment (Jonkrua) [reply projects/developer_tooling/3Web-DAO.md:1]: add a project name done

File: projects/developer_tooling/3Web-DAO.md (hunk 2 above)
Comment (Jonkrua) [reply projects/developer_tooling/3Web-DAO.md:4]: move this markdown file to application folder done

File: projects/developer_tooling/3Web-DAO.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+
+## Project Description
+
+3Web DAO is Decentralized Autonomous Organization in crypto.It contains 
+3Web Analytics(similar glassnode/defillama): focusing on data analysis on the blockchain; 
+3Web Research(similar messari): research institution focusing on the blockchain crypto field;
+3Web DAO's vision is to help unicorn project and investor in Web3.
+
+We all know that Filecoin community developers are waiting for the release of the much-anticipated FVM. It is believed that FVM will bring more users and traffic to the Filecoin ecosystem.So 3Web DAO will analysis on-chain data in Filecoin and output research reports to provide Filecoin developers and investors with more professional data and reports in the next period of time.
```
[the remark this answers is not in the document]
Comment (Jonkrua) [reply projects/developer_tooling/3Web-DAO.md]: please see the `Project Description` on new 3WebDAO.md

File: projects/developer_tooling/3Web-DAO.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+https://github.com/3Web-DAO
+
+## Website
+
+https://3web.io/  (Coming soon...The domain name has been purchased and website is under development)
+
+## Docs
+
+https://mirror.xyz/0x8C5da2937C664882111FCfe717ca7AF02A6e6440
```
[the remark this answers is not in the document]
Comment (Jonkrua) [reply projects/developer_tooling/3Web-DAO.md]: keep it blank now

**Review by longfeiWan9 — APPROVED** [review]

**Pull request merged**